docs/dev/news/README.md

```diff
@@ -0,0 +1,23 @@
+# Latest Dafny changes
+
+Each file in this folder describes one change in Dafny since the latest release.
+
+Files in this directory are named `<PR number>.<kind>` and each contains release notes for one merged PR.  `kind` is `feat` or `fix`.  No need to include a link to the PR in the file: it will be added automatically.
+
+## Examples
+
+`1234.fix`
+
+```
+Improved toaster settings prevent Dafny from ever burning toast.
+Instead, toast will always come out crispy and nice
```

Comment:
If you don't like my comment above and still find that multi-line release notes are relevant, like, we want to output more user-centric release notes that not only have a title, but a small example or explanation:

```suggestion
Added support for :==, like this:
    method Test(x: int, y: int) {
      var x := 1;
      x :== 1; // Assertion succeeds.
      x :== 2; // Assertion fails.
    }
```

Then, in this case, if it's multiline, you can add a newline (no automatically inserted period) and put the link in one more link. But if someone writes only one line, then if there is a period, then you move it after the (link) that is inserted in the sentence.